File: 1-Iniciante/1047-Tempo-de-Jogo-com-Minutos/1047.c
#include <stdio.h>

int main() {
    int time[4];
    scanf ("%d %d %d %d", &time[0], &time[1], &time[2], &time[3]);
    if (time[1] > time[3]) {
        time[2] -= 1;
        time[3] += 60;
    }
    if (time[0] < time[2] || time[0] == time[2] && time[1] < time[3]) {
        printf ("O JOGO DUROU %d HORA(S) E %d MINUTOS(S)\n", time[2] - time[0], time[3] - time[1]);
    }
    else {
        printf ("O JOGO DUROU %d HORA(S) E %d MINUTOS(S)\n", time[2] - time[0] + 24, time[3] - time[1]);
    }
    return 0;
}

File: 1-Iniciante/1047-Tempo-de-Jogo-com-Minutos/1047.py
time = {}
x = input().split()
for i, _ in enumerate(x):
    time.update({i : int(_)})
if time[1] > time[3]:
    time[2] -= 1
    time[3] += 60
if time[0] < time[2] or time[0] == time[2] and time[1] < time[3]:
    print ("O JOGO DUROU", time[2] - time[0], "HORA(S) E", time[3] - time[1], "MINUTO(S)")
else:
    print ("O JOGO DUROU", time[2] - time[0] + 24, "HORA(S) E", time[3] - time[1], "MINUTO(S)")